frontend/src/Components/Shared/RepoTable/RepoTable.tsx

```diff
@@ -110,7 +110,7 @@ const RepoTable = ({
 
   // augment / build columns from schema (and add delete action)
   useEffect(() => {
-
+    let isMounted = true;
```

Comment:
just a suggestion. This `isMounted` is basically saying the component is mounted or not. Correct me if I am wrong, but the component is considered mounted only after all the information has been fetched, right?
If that is the case, I would suggest change this variable to something more familiar like `isLoading` or `isLoading[iname of information being loaded]`, since that is the name being used in other components and is more obvious in my opinion.

Reply:
I just learned about this this week :-)

Fetching information in a side-effect is initializing, not mounting. The component will be mounted before side-effects are run to finish the initialization.

The method returned by useEffect is the equivalent of `componentWillUnmount' in React class-based components. This means that our handler will set `isMounted` to `false` when React decides to unmount the component. https://reactjs.org/docs/hooks-effect.html

I believe it's important to be totally on the same page with the basics here, so let's discuss this further until we're both satisfied and 100% sure that we understand what we're doing here.